client/mycart/src/app/user/register/register.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="container-fluid bg bg-info  d-flex justify-content-center align-items-center">
+    <div class="container">
+        <h2 class="text-center text-warning mb-4 pt-4">REGISTER</h2>
+        <div class="row">
+            <div class="col-md-6 offset-md-3">
+                <form #form="ngForm" (ngSubmit)="onSubmit(form)">
+                    <div class="my-3">
+                        <label>FullName</label>
+                        <input type="text" name="fname" [(ngModel)]="model.fname" #fname="ngModel" class="form-control"
+                            placeholder="Enter FullName" maxlength="30" required>
+                        <div class="text-danger" *ngIf="fname.invalid && fname.touched">
+                            <div *ngIf="fname.errors?.['required']">FullName is required</div>
+                        </div>
+                    </div>
+                    <div class="my-3">
+                        <label>Email ID</label>
+                        <input type="email" name="email" [(ngModel)]="model.email" #email="ngModel" class="form-control"
+                            placeholder="Enter Email ID" minlength="5" maxlength="70"
+                            pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required>
+                        <div class="text-danger" *ngIf="email.invalid && email.touched">
+                            <div *ngIf="email.errors?.['required']">Email is required</div>
+                            <div *ngIf="email.errors?.['minlength']">Email is too short</div>
+                            <div *ngIf="email.errors?.['maxlength']">Email is too long</div>
+                            <div *ngIf="email.errors?.['pattern']">Email is Invalid</div>
+                        </div>
+                    </div>
+                    <div class="my-3">
+                        <label>Contact Number</label>
```

Comment:
Can you change the text to phone number?
```suggestion
<label>Phone Number</label>
```